File: 1360-number-of-days-between-two-dates/1360-number-of-days-between-two-dates.java
import java.time.LocalDate;
import java.time.temporal.ChronoUnit;

class Solution {
    public int daysBetweenDates(String date1, String date2) {
        LocalDate a = LocalDate.parse(date1);
        LocalDate b = LocalDate.parse(date2);
        return (int)Math.abs(ChronoUnit.DAYS.between(a, b));
    }
}

File: 1360-number-of-days-between-two-dates/1360-number-of-days-between-two-dates.kt

import java.time.Duration
import java.time.LocalDate
import java.time.format.DateTimeFormatter

class Solution {
fun daysBetweenDates(date1: String, date2: String): Int = Math.abs(Duration.between(LocalDate.parse(date2, DateTimeFormatter.ofPattern("yyyy-MM-dd")).atStartOfDay(), LocalDate.parse(date1, DateTimeFormatter.ofPattern("yyyy-MM-dd")).atStartOfDay()).toDays().toInt())
}